cfn-resources/alert-configuration/cmd/resource/resource.go

```diff
@@ -158,53 +166,55 @@ func Update(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		currentModel.Profile = aws.String(profile.DefaultProfile)
 	}
 
-	client, peErr := util.NewMongoDBClient(req, currentModel.Profile)
+	client, peErr := util.NewAtlasClient(&req, currentModel.Profile)
 	if peErr != nil {
 		return *peErr, nil
 	}
+	atlasV2 := client.AtlasV2
 
 	// Check if  already exist
-	if !isExist(currentModel, client) {
+	if !isExist(currentModel, atlasV2) {
 		_, _ = logger.Warnf("resource not exist for Id: %s", *currentModel.Id)
 		return handler.ProgressEvent{
 			OperationStatus:  handler.Failed,
 			Message:          "Resource Not Found",
 			HandlerErrorCode: cloudformation.HandlerErrorCodeNotFound}, nil
 	}
-	var res *mongodbatlas.Response
 
 	// In order to update an alert config it is necessary to send the original alert configuration request again, if not the
 	// server returns an error 500
 	projectID := *currentModel.ProjectId
 	id := *currentModel.Id
-	alertReq, res, err := client.AlertConfigurations.GetAnAlertConfig(context.Background(), projectID, id)
+	alertReq, res, err := atlasV2.AlertConfigurationsApi.GetAlertConfiguration(context.Background(), projectID, id).Execute()
```

Comment:
please `close()` here

Reply:
we don't need to close it as this is done in the SDK https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_5_best_practices.md#linting-issues

This is an error with the Lint so we should update `.golangci.yml` to ignore `bodyclose`

Reply:
Thanks @andreaangiolillo I will update .golangci in that case and remove body.close()

Reply:
updated